Service/OasService.php

```diff
@@ -15,13 +15,21 @@ class OasService
      */
     private EntityManagerInterface $entityManager;
 
+    /**
+     * @var ParameterBagInterface
+     */
+    private ParameterBagInterface $parameters;
```

Comment:
You're using parameterBag to access this variable, ParameterBagInterface is missing a 'use' statement.